components/ApiUserDashboard.js

```diff
@@ -0,0 +1,71 @@
+import { Component } from 'react'
+import { Alert, Button } from 'react-bootstrap'
+
+import ApiKeyList from './ApiKeyList'
+import RequestLogsDashboard from './RequestLogsDashboard'
+import {OTP_PLAN_URL} from '../util/constants'
+
+/**
+ * The high-level component visible to an ApiUser when they log in. This
+ * describes the user's API keys, points to some docs, and shows API key usage.
+ */
+class ApiUserDashboard extends Component {
```

Comment:
Are you asking if this can be a functional component? If so, it probably can be. I don't really understand the actual benefit of making that code change though.